www/config.yml

```diff
@@ -1369,3 +1369,13 @@ subtitle: Women Who Love Coding in Python
 tag_layout: tag.html
 tags_url: /tags/
 title: PyLadies
+email: singapore@pyladies.com
+image: pyladies-sg
```

Comment:
Did you upload the file? It should be added to /www/_assets/images directory

And this should be a filename i.e pyladies-sg.jpeg or something like that. 
Please see examples from previous entries above.